### PR TITLE
Add Decal icons

### PR DESCRIPTION
**Pull request by StrawberryMoses**
## About the PR 
Adds some icons from TG for use with decals, such as cleanables like ash or tile overlays. Adds additional items that were in the same resource packs. Don't have the skill level yet to tackle something as big as overlays

### CODE REVIEW

**Comment by PJB3005** [comment]
Don't bother putting this stuff in the changelog if it's not player visible.

**Comment by StrawberryMoses** [comment]
wasnt aware, woops
removed

File: Resources/Textures/Constructible/Misc/furniture.rsi/meta.json [hunk 1]
```diff
@@ -1,12 +1,21 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from cev-eris at https://github.com/discordia-space/CEV-Eris/commit/59fe5dd2841f47a8abce60eecb9fafad34282bd0",
+  "copyright": "Taken from cev-eris and tgstation at https://github.com/discordia-space/CEV-Eris/commit/59fe5dd2841f47a8abce60eecb9fafad34282bd0 and https://github.com/tgstation/tgstation/commit/2528a7fd89dca1369da7ce56378e6cb2ddd01161",
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Misc/furniture.rsi/meta.json:4]: This is vague on which ones are which.

File: Resources/Textures/Constructible/Misc/furniture.rsi/meta.json [hunk 2]
```diff
@@ -1,12 +1,21 @@
...
     "y": 32
   },
   "states": [
+    {
+      "name": "abed",
+      "directions": 1,
+      "delays": [
+        [
+          1.0
+        ]
```
Comment (metalgearsloth) [inline Resources/Textures/Constructible/Misc/furniture.rsi/meta.json:16]: Don't need these if they're defaults.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
Swept loves scripted imports.

**Comment by SweptWasTaken** [comment]
I don't agree with putting RSIs in the game without a purpose. Also these are scripted and don't have correct formatting.  I recommend you upload these to the [RSI Dump](https://github.com/space-wizards/rsi-dump) for the person who adds decals proper.

**Pull request closed without merging**